components/details/detail.html

```diff
@@ -15,10 +15,23 @@ <h5 class="modal-title" id="staticBackdropLabel">Product: {{$ctrl.product[0].nam
                 <button type="button" class="btn btn-secondary" data-bs-dismiss="modal" ng-click="$ctrl.onBack()">Go
                     Back
                 </button>
-                <button type="button" class="btn btn-primary" ng-click="$ctrl.onAddToCart($ctrl.product[0])">Add
+                <button id="liveToastBtn" type="button" class="btn btn-primary"
+                        ng-click="$ctrl.onAddToCart($ctrl.product[0])">Add
                     to Cart
                 </button>
             </div>
         </div>
+        <div class="position-fixed bottom-0 end-0 p-3">
+            <div id="liveToast" class="toast hide" role="alert" aria-live="assertive" aria-atomic="true">
+                <div class="toast-header">
+                    <strong class="me-auto">Success</strong>
+                    <small>{{$ctrl.interval}} seconds ago</small>
```

Comment:
How many e-commerce websites did you use that had this idea of "added x seconds ago"? :) I'm just saying that maybe we could get rid of this "interval" feature...

Reply:
I agree